### PR TITLE
Create new historical production parser for Chile

### PR DESCRIPTION
**Pull request by systemcatch**
TODO
- [x] Docstrings
- [x] Configs (zone, locales)
- [x] Readme
- [x] New world.json
- [x] Check and bump any requirements needed
- [x] Mark "dead" parsers

ref #655

### CODE REVIEW

File: config/zones.json [hunk 1]
```diff
@@ -800,70 +800,38 @@
     },
     "timezone": null
   },
-  "CL-SIC": {
+  "CL-SING-SIC": {
```
Comment (corradio) [inline config/zones.json]: from a quick Google search it seems like this is now merged into a zone called "SING". I might be wrong but I think SIC doesn't exist anymore.

File: parsers/CL_SIC.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python3
 
-"""Parser for the SIC grid region of Chile."""
+"""Parser for the SIC grid region of Chile.
```
Comment (corradio) [inline parsers/CL_SIC.py]: let's just delete that file

File: parsers/US_SVERI.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python3
 
-"""Parser for the SVERI area of the USA."""
+"""Parser for the SVERI area of the USA.
```
Comment (corradio) [inline parsers/US_SVERI.py]: same here

File: web/locales/es.json [hunk 4]
```diff
@@ -320,13 +320,9 @@
             "countryName": "Chile",
             "zoneName": "SEM"
         },
-        "CL-SIC": {
+        "CL-SING-SIC": {
```
Comment (corradio) [inline web/locales/es.json]: good call 👍

**Comment by systemcatch** [comment]
> Thank you for keeping this alive :)

Not a problem, it's good to have you back Oli.

**Pull request merged**